deno/send_message/src/functions/send_tweet.ts

```diff
@@ -0,0 +1,54 @@
+import { statusUpdate } from "https://deno.land/x/twitter_api_client/api_v1/tweets/update.ts";
+
+export default async function (variables: any, text: string) {
+  const consumerKey = variables["TWITTER_API_KEY"];
+  const consumerSecret = variables["TWITTER_API_KEY_SECRET"];
+  const accessToken = variables["TWITTER_ACCESS_TOKEN"];
+  const accessTokenSecret = variables["TWITTER_ACCESS_TOKEN_SECRET"];
+
+  if (!consumerKey) {
+    return {
+      success: false,
+      message: "TWITTER_API_KEY is not set",
+    };
+  }
+  if (!consumerSecret) {
+    return {
+      success: false,
+      message: "TWITTER_API_KEY_SECRET is not set",
+    };
+  }
+  if (!accessToken) {
+    return {
+      success: false,
+      message: "TWITTER_ACCESS_TOKEN is not set",
+    };
+  }
+  if (!accessTokenSecret) {
+    return {
+      success: false,
+      message: "TWITTER_ACCESS_TOKEN_SECRET is not set",
+    };
+  }
+
+  const res = await statusUpdate(
```

Comment:
It seems some error is causing:

```json
{
  "stdout": "{\"variables\":{\"TWITTER_API_KEY\":\"asdf\",\"TWITTER_API_KEY_SECRET\":\"asdf\",\"TWITTER_ACCESS_TOKEN\":\"asdf\",\"TWITTER_ACCESS_TOKEN_SECRET\":\"asdf\"},\"headers\":{},\"payload\":{\"type\":\"Twitter\",\"receiver\":\"hello@example.com\",\"message\":\"Programming is fun!\",\"subject\":\"Programming is funny!\"}}\n{}",
  "stderr": "Code file not found."
}
```

with the following request:

```shell
curl -H "X-Internal-Challenge: secret-key" \
    -H "Content-Type: application/json" \
    -X POST http://localhost:3000/ \
    -d '{
    "variables": {
        "TWITTER_API_KEY": "asdf",
        "TWITTER_API_KEY_SECRET": "asdf",
        "TWITTER_ACCESS_TOKEN": "asdf",
        "TWITTER_ACCESS_TOKEN_SECRET": "asdf"
    },
    "payload": {
        "type": "Twitter",
        "receiver": "hello@example.com",
        "message": "Programming is fun!",
        "subject": "Programming is funny!"
    }
}'
```

Reply:
@Rinshin-Jalal this exact command is still resulting in the unhelpful error message.